include/ros1_bridge/bridge.hpp

```diff
@@ -18,6 +18,7 @@
 #include <map>
 #include <memory>
 #include <string>
+#include <utility>
```

Comment:
This one seems odd, I didn't see it in the test out and you didn't modify it otherwise, so it should have been failing before.